src/api/common/createApi.ts

```diff
@@ -0,0 +1,23 @@
+import axios, { AxiosInstance, AxiosResponse, AxiosRequestConfig } from 'axios'
```

Comment:
It's better to avoid long lines in code:

```suggestion
import axios, {
  AxiosInstance,
  AxiosResponse,
  AxiosRequestConfig,
} from 'axios'
```